src/frontends/tensorflow_common/src/op/binary_op.cpp

```diff
@@ -142,6 +142,31 @@ OutputVector translate_mul_op(const NodeContext& node) {
     set_node_name(node.get_name(), result);
     return {result};
 }
+
+OutputVector translate_addv2_op(const NodeContext& node) {
+    default_op_checks(node, 2, {}, true);
+    auto lhs = node.get_input(0);
+    auto rhs = node.get_input(1);
+    auto result = make_shared<v1::Add>(lhs, rhs);
+
+    auto complex_type_mark_lhs = as_type_ptr<ComplexTypeMark>(lhs.get_node_shared_ptr());
+    auto complex_type_mark_rhs = as_type_ptr<ComplexTypeMark>(rhs.get_node_shared_ptr());
+    if (complex_type_mark_lhs || complex_type_mark_rhs) {
+        FRONT_END_GENERAL_CHECK(complex_type_mark_lhs != nullptr && complex_type_mark_rhs != nullptr,
+                                "AddV2 got complex and non-complex inputs. Inputs should be of the same type.");
```

Comment:
```suggestion
    if (complex_type_mark_lhs && complex_type_mark_rhs) {
```

Let assume that the graph is valid and consistent.